sdk/README.md

```diff
@@ -329,14 +329,20 @@ This is achieved with migrations.
 
 ##### SQLite
 
-TBD.
+Postgresql migrations live in `./component/store/local/sqlc/sqlc/migrations`.
+
+In order to create a new migration, you can follow these steps:
+
+* run `$ make new-sqlite-migration migration_name=my_migration`
+* edit the migration
+* run `$ update-sqlite-migrations-sum` to update `atlas.sum`
```

Comment:
What is `altlas.sum` and why is it needed? Is that the migration manager?

Reply:
It's used by atlas to understand if the generated migrations are valid or they were manually changed

Reply:
Should that be in the readme?

Reply:
Can add in the next PR!